controllers/propagator/propagation.go

```diff
@@ -782,6 +811,19 @@ func (r *PolicyReconciler) handleDecision(instance *policiesv1.Policy, decision
 		// template delimis detected, build a temp holder policy with templates resolved
```

Comment:
Optional: I know you didn't make this change, but it'd be easier to read if you changed `if policyHasTemplates(instance) {` to `if policyHasTemplates(comparePlc) {`.